java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -289,6 +251,115 @@
     // grid size in pixels
     private int gridSize = 10;
 
+
+    // size of point boxes
+    @Deprecated
+    public static final double SIZE = 3.0;
+    @Deprecated
+    public static final double SIZE2 = SIZE * 2.;  // must be twice SIZE
+
+    // defined constants - turnout types
+    @Deprecated
+    public static final int RH_TURNOUT = LayoutTurnout.RH_TURNOUT;
```

Comment:
The formerly public fields also need a JavaDoc reading at least something similar to:
```
/**
 * @deprecated since 4.7.1; use {@link jmri.jmrit.display.layoutEditor.LayoutTurnout#RH_TURNOUT} instead
 */
```
This JavaDoc provides both:
- documentation of what the deprecated field is replaced with
- a clue as to when we can remove the deprecated field (not before two production releases after 4.8 in this case)